qrenderdoc/Windows/TextureViewer.cpp

```diff
@@ -759,12 +759,12 @@ void TextureViewer::RT_FetchCurrentPixel(IReplayController *r, uint32_t x, uint3
     }
   }
 
-  realValue = r->PickPixel(id, x, y, sub, CompType::Typeless);
+  realValue = r->PickPixel(id, x, y, sub, typeCast);
 
   if(m_TexDisplay.customShaderId != ResourceId())
   {
     pickValue = r->PickPixel(m_Output->GetCustomShaderTexID(), x, y,
-                             {m_TexDisplay.subresource.mip, 0, 0}, CompType::Typeless);
+                             {m_TexDisplay.subresource.mip, 0, 0}, typeCast);
```

Comment:
We shouldn't apply the typecast for picking custom shader output, since that's rendered to a fixed float buffer and isn't affected by any view typecasting.

Reply:
Fixed